### PR TITLE
feat: [RPL-PS] Upstream RPL-PS code.

### PR DESCRIPTION
**Pull request by bejeanmo**
Add RPL-PS Platform code to public repo.

### CODE REVIEW

File: Platform/RaptorlakeBoardPkg/BoardConfigRplPs.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+##
+# Import Modules
+#
+import os
+import sys
+#tool_dir = os.path.realpath(os.path.join(os.path.dirname (os.path.realpath(__file__)), '..', '..', '..', 'SblOpen', 'Platform', 'AlderlakeBoardPkg'))
+#sys.path.append (tool_dir)
```
Comment (gdong1) [inline Platform/RaptorlakeBoardPkg/BoardConfigRplPs.py:15]: Remove these commented code.

File: Platform/RaptorlakeBoardPkg/Rplps/Microcode/Microcode.inf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+[Defines]
+  INF_VERSION          = 0x00010005
+  BASE_NAME            = Microcode
+  FILE_GUID            = 197DB236-F856-4924-90F8-CDF12FB875F3
+  MODULE_TYPE          = USER_DEFINED
+  VERSION_STRING       = 1.0
+
+[Sources]
+m_e0_b06a2_00004121.mcb    # RPL-PS
+m_80_906a3_00000429.mcb    # ADL-P/M Lx/Rx
```
Comment (tsaikevin) [inline Platform/RaptorlakeBoardPkg/Rplps/Microcode/Microcode.inf:18]: @bejeanmo do we need this m_80_906a3_00000429.mcb?

File: Silicon/RaptorlakePkg/Rplps/Fsp/FspBinRplps.inf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#
+##
+
+
+[UserExtensions.SBL."CloneRepo"]
+  REPO    = https://github.com/intel/FSP.git
+  COMMIT  = 0efd8a3723568012e8ac2ccc86f6beef1b440230
+
+[UserExtensions.SBL."CopyList"]
+# For RPL-P
```
Comment (tsaikevin) [inline Silicon/RaptorlakePkg/Rplps/Fsp/FspBinRplps.inf:17]: @bejeanmo fix the board name, it's "RPL-PS"

**Review by gdong1 — APPROVED** [review]
looks good to me in general.

File: Silicon/RaptorlakePkg/Rplps/Fsp/FspBinRplps.inf (hunk 3 above)
Comment (bejeanmo) [reply Silicon/RaptorlakePkg/Rplps/Fsp/FspBinRplps.inf]: Good catch Kevin. Should be fixed.

File: Platform/RaptorlakeBoardPkg/Rplps/Microcode/Microcode.inf (hunk 2 above)
Comment (bejeanmo) [reply Platform/RaptorlakeBoardPkg/Rplps/Microcode/Microcode.inf]: Thanks. I thought I had removed these already. Should be fixed now.

**Review by gdong1 — APPROVED** [review]

**Review by tsaikevin — APPROVED** [review]

**Review by tsaikevin — APPROVED** [review]

**Pull request merged**